cleaner.py

```diff
@@ -144,55 +143,56 @@ def clean_command(text: str):
     username = text[username_index:username_end_index]
 
     command_index = text.find(":", username_end_index) + 2
-    end_of_the_line = text.__len__() - 1
+    end_of_the_line = len(text) - 1
 
     clean_text = text[:time_end] + "!!! CMD " + username + " issued /" + text[command_index:end_of_the_line] + " (success)\n"
     return clean_text
 
 
 raw_log = input("Enter log file path: ")
-if raw_log[raw_log.__len__() - 4:] != ".log":
+if not raw_log.endswith(".log"):
     print("ERROR: This isn't even a log file! Enter a valid one.")
     print("Exiting program...")
     exit()
 
 print("Opening files...")
 
-filter = open("filters.json", "r")
-filter = filter.read() # Read file as plaintext
-filter = json.loads(filter) # Turn plaintext into dictionary
+filter = {}
+with open("filters.json", "r") as file:
+    filter = file.read() # Read file as plaintext
+    filter = json.loads(filter) # Turn plaintext into dictionary
 
 raw_log = open(raw_log, "r")
 cleaned_log = open("cleaned.log", "w")
 
 raw_lines = raw_log.readlines()
 cleaned_lines = [
-    "Log cleaned at {}\n".format(datetime.datetime.now())
+    "Log cleaned at {}\n".format(datetime.now())
 ]
 
 print("Begin filtration")
 
 players_login_times = {}
 last_chatter = ""
 day_timestamp = ""
-iteration = 0
 keep_last_chatter = False
 for line in raw_lines:
     # check if the line should be filtered out
     found_illegal = False
     for item in filter:
         low_line = line.lower()
-        if low_line.__contains__(item.lower()) or check_for_IP_at_index(low_line, raw_line_start):
+        if item.lower() in low_line or check_for_IP_at_index(low_line, raw_line_start):
             found_illegal = True
+            break
```

Comment:
This exits the loop as soon as a hit is found. Makes the program a tiny bit faster.